sdk/core/Azure.Core/api/Azure.Core.net5.0.cs

```diff
@@ -569,3 +570,29 @@ public abstract partial class ObjectSerializer
         public virtual System.Threading.Tasks.ValueTask<System.BinaryData> SerializeAsync(object? value, System.Type? inputType = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
 }
+namespace Azure.Messaging
+{
+    [System.Text.Json.Serialization.JsonConverterAttribute(typeof(Azure.Messaging.CloudEventConverter))]
+    public partial class CloudEvent
+    {
+        public CloudEvent(string source, string type, object jsonSerializable, System.Type dataSerializationType = null) { }
+        public CloudEvent(string source, string type, System.ReadOnlyMemory<byte> data, string dataContentType) { }
+        public string DataContentType { get { throw null; } set { } }
+        public string DataSchema { get { throw null; } set { } }
+        public System.BinaryData EventData { get { throw null; } }
```

Comment:
Not sure I like EventData over Data, considering there are other properties that are prefixed with Data - DataContentType, DataSchema

Reply:
/cc @tg-msft as he didn't like `Data`

Reply:
I think we should call it Data. https://github.com/cloudevents/spec/blob/v1.0.1/spec.md#data